wpeverest/user-registration
# UR-231 Feature - Recaptcha v2 invisible and threshold score

### All Submissions:






### Changes proposed in this Pull Request:



Previously, We do not have features like threshold score checking in Recaptcha version v3 and Invisible Recaptcha version v2. This PR aims to fix these issues.
### How to test the changes in this Pull Request:

1. Navigate to the User registration settings and then the Integration tab. 
2. now you can see the option for invisible Recaptcha in v2 and threshold score option for v3.
3. enable the invisible Recaptcha and add the site and secret key and verify whether it is working as expected or not.

### Types of changes:




### Other information:




### Changelog entry

> Feature - Recaptcha v2 invisible and threshold score.
